ptero_workflow/api/v1/views.py

```diff
@@ -44,6 +45,26 @@ def get(self, workflow_id):
             LOG.exception('Unexpected exception getting workflow')
             raise
 
+class ExecutionInputsView(Resource):
+    def get(self, execution_id):
+        try:
+            execution_input_data = g.backend.get_execution_inputs(execution_id)
+            return execution_input_data, 200
+        except:
+            LOG.exception('Unexpected exception getting execution inputs')
+            raise
+
+class ExecutionOutputsView(Resource):
+    def put(self, execution_id):
+        try:
+            g.backend.set_execution_outputs(execution_id, request.get_json())
+            return '', 200
+        except OutputsAlreadySet:
+            return '', 403
```

Comment:
Although "Conflict" seems to describe well what is going on the [rfc](http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.10) says that 409 should only be used in cases where the request may be repeated.  If we're going to forbid setting outputs twice, then the request should not be repeated, and 409 should not be used.